users/models/report.py

```diff
@@ -33,6 +44,31 @@ async def handle_new(cls, instance: "Report"):
                 )
             except httpx.RequestError:
                 return
```

Comment:
Let's no longer return here, but `pass` - if we can't send the report to the other server, I'd rather we just get the email rather than trying forever.